application/backend/src/main/resources/apps/typerefinery/components/clientlibs/clientlibs-header/components.js

```diff
@@ -32,5 +32,414 @@ window.Typerefinery.VueData = Typerefinery.VueData || {};
     ns.getQueryParams = () => {
         const query = window.location.search.substring(1);
         return ns.queryToObject(query);
-    }
+    };
+    ns.fireballSvg = `
```

Comment:
can we do this as an asset link rather than copy this everywhere on the page?

Reply:
yeah sure